internal/sgx/sgx.go

```diff
@@ -357,6 +382,11 @@ func (ctx *SgxContext) provisionKey(signerName string, wrappedSWK []byte, wrappe
 	ctx.ctxLock.Lock()
 	defer ctx.ctxLock.Unlock()
 
+	quoteInfo := ctx.quotes[signerName]
```

Comment:
Maybe follow the same `quoteInfo, ok :=` style as above?

Reply:
Done.